data/themes/default/style.css

```diff
@@ -620,15 +620,17 @@ TrackContentObjectView   {
 	qproperty-selectedColor: #006B65;
 	qproperty-BBPatternBackground: #373d48;
 	qproperty-textColor: #fff;
+	qproperty-textBackgroundColor: rgba(0, 0, 0, 75);
 	qproperty-textShadowColor: rgb(0,0,0,200);
```

Comment:
@tresf Ah, ok. It looks like this:

![3704-colored-bb-patterns](https://user-images.githubusercontent.com/9293269/28336115-13f1686e-6c01-11e7-8d11-c971fde3bda3.png)